src/StringSearcher.java

```diff
@@ -1,4 +1,41 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
 
 public class StringSearcher {
+	private String word;
+	private String fileName;
 
```

Comment:
Please ensure to comment your code so that future employees can use your function.